lib/mobility-core/src/Kernel/External/Verification/Interface.hs

```diff
@@ -58,6 +69,20 @@ validateImage ::
   m ValidateImageResp
 validateImage serviceConfig req = case serviceConfig of
   IdfyConfig cfg -> Idfy.validateImage cfg req
+  FaceVerificationConfig _ -> throwError $ InternalError "Not Implemented!"
+
+validateFaceImage ::
+  ( MonadThrow m,
+    CoreMetrics m,
+    MonadIO m,
```

Comment:
`Kernel.Types.App.MonadFlow` already contains `MonadIO` and `MonadThrow`, hence you can remove them from here.

Reply:
same for `validateFace` function